index.html

```diff
@@ -182,6 +182,7 @@ <h2 class="cntct-ttl">Contact me</h2>
       <div id="project-details" class="display-none project-details"></div>
       
       <script src="index.js" type="module"></script>
+      <script src="./validateForm.js"></script>
       <!-- <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
         <script src="https://cdn.jsdelivr.net/npm/popper.js@1.14.7/dist/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
         <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script> -->
```

Comment:
- [ ] Kindly avoid committing old pieces of code as inline comments. They make your project look messy. If you need to review a previous version of your code, you can always use git history :100: . So please remove all the commented pieces of code from your project :+1: